functions/system_to_ip_map.py

```diff
@@ -0,0 +1,27 @@
+def system_to_ip_map(sys_names,ips):
+	"""function to map system with ip address and return as dictionary.
+    Args:
+       sys(tuple): The first parameter -system names
+       ips(tuple): The second parameter -ip address of system
+
+    Returns:
+        Dict: The return value, ordered dictionary with key - system name and value - ip address.
+
+    Example:
+    	sys_names='system1','system2'
+    	ips='192.168.1.2','192.168.1.3'
+
+    	[('system1', '192.168.1.2'), ('system2', '192.168.1.3')]
+    """	
+	ip_mapped_dict=zip(sys_names,ips)
```

Comment:
My intention of giving this problem you will implement zip functionality, if you are using zip then no need of this function itself.

Reply:
Remove zip and implement your own logic here.

Reply:
removed zip and directly printing the output